src/dotnet/HQ.Server/Data/Models/Staff.cs

```diff
@@ -7,7 +7,7 @@ public class Staff : Base
     public string Name { get; set; } = null!;
     public string? FirstName { get; set; }
     public string? LastName { get; set; }
-    public string? Email { get; set; }
+    public string Email { get; set; } = null!;
```

Comment:
Leave the Email property nullable on the database model, this is used by EF to infer schema changes and we don't want to retroactively make it non-nullable. Going forward it would be handled by the validation.